conf/simconfig.toml

```diff
@@ -29,3 +29,6 @@ leader-schedule-limit = 32
 region-schedule-limit = 128
 replica-schedule-limit = 32
 merge-schedule-limit = 32
+## TODO:Find a better place to put this config
```

Comment:
Seem we don't need to add these configurations here.

Reply:
Yes, we can remove it in simconfig.